### PR TITLE
improvement: increase distinct look of hidden columns

### PR DESCRIPTION
**Pull request by brandstetterm**
## Description

After it is repeatedly noted that it is not quite clear whether a column is hidden or not, I've tried to make some improvements.

## Changelog
- Add crosshatched background to hidden columns
- Change the color from the hidden icon from grey to red

## (Optional) Visual Changes

Screen Shot 2022-12-08 at 10 22 32
Screen Shot 2022-12-08 at 10 22 25
Screen Shot 2022-12-08 at 10 21 57
Screen Shot 2022-12-08 at 10 21 41

### CODE REVIEW

File: src/components/Column/Column.scss [hunk 1]
```diff
@@ -172,7 +180,7 @@
   min-width: $icon--medium;
   min-height: $icon--medium;
   margin: 0 $margin--small 0 0;
-  color: $color-middle-gray;
+  color: $color-warning-red;
```
Comment (CronJorian) [inline src/components/Column/Column.scss:183]: image

What do you think about a border around the icon?

Comment (CronJorian) [reply src/components/Column/Column.scss:183]:
```suggestion
  color: $color-warning-red;
  border: 2px solid $color-warning-red;
  box-sizing: content-box;
  border-radius: $border-radius--round;
```

Comment (CronJorian) [reply src/components/Column/Column.scss:183]: Oh and in case you want to use this ^, you should probably add the border-color to the hover effect, too :)

Comment (brandstetterm) [reply src/components/Column/Column.scss:183]: Ah, I think it look's better and more minimalistic without the border 😄

**Pull request merged**